treeherder/model/models.py

```diff
@@ -210,14 +210,14 @@ def __str__(self):
 
 @python_2_unicode_compatible
 class Bugscache(models.Model):
-    id = models.AutoField(primary_key=True)
-    status = models.CharField(max_length=64, blank=True, db_index=True)
+    id = models.PositiveIntegerField(primary_key=True)
```

Comment:
Of course, you're absolutely right. Sorry for the confusion. ;)

FWIW if I were designing this table myself from scratch I would probably not make the bug id the primary key.